### DISCUSSION TITLE
Stone remake of wooden pieces

### DISCUSSION THREAD

**Discussion by Martint-T** [BentoGambin/Valheim-MissingPieces#1] [Ideas]
Its a wonderful mod, but some more structures are missing yet.
There are a mod for a stonevariant from all woodpieces, but they dont have the same textures.
So maybe you can add these pieces with the same textures at your mod :)

**Comment by BentoGambin** [comment]
Hi Martint. Thanks for the suggestion. If I'm going to be honest with you, whole piece groups re-do's are not the intention of this mod. This sucker is meant to only add those pieces that should be in vanilla but aren't. Leave those overhauls for cookie to do... [😂](https://emojipedia.org/face-with-tears-of-joy/)
